app/views/certificates/index.html.erb

```diff
@@ -26,6 +28,8 @@
 
 <td><%= certificate.owner_id %></td>
 
+<%# CRUD operations %>
+
```

Comment:
h

Reply:
Include the complete App Name instead of CRUD operations tag. https://github.com/Usama-Naeem/Employee-App/issues/4

Reply:
Issue has been solved